samples/msteams-virtual-assistant/csharp/.filenesting.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could you please rename this folder to match other samples? Could you please change it to bot-virtual-assistant?

Reply:
It is a sample app demonstrating how virtual assistant can be configured to use msteams platform capabilities. It has a bot, task module and compose extension. Hence don't want to name it bot specific.

Reply:
Sure. Could you please then change it to app-virtual-assistant? Please change the hyperlinks present in root readme file as well.